mail/mail-policy.go

```diff
@@ -1,27 +1,37 @@
 package mail
 
 import (
+	"fmt"
+
 	"github.com/wopta/goworkspace/models"
 )
 
-func GetMailPolicy(policy models.Policy, subject string, islink bool, link string, linkLabel string, lines []string, isAttachment bool, at *[]Attachment) MailRequest {
-	var name string
-	var linkForm string
-	if policy.Name == "pmi" {
-		name = "Artigiani & Imprese"
+func GetMailPolicy(
+	policy models.Policy,
+	subject string,
+	isLink bool,
+	link string,
+	linkLabel string,
+	lines []string,
+	isAttachment bool,
+	at *[]Attachment,
+) MailRequest {
+	var (
+		name     string
+		obj      MailRequest
 		linkForm = "https://www.wopta.it/it/multi-rischio/"
+	)
 
-	}
-	if policy.Name == "persona" {
+	switch policy.Name {
+	case "pmi":
+		name = "Artigiani & Imprese"
+	case "persona":
 		name = "Persona"
-		linkForm = "https://www.wopta.it/it/multi-rischio/"
-	}
-	if policy.Name == "life" {
+	case "life":
 		name = "Vita"
-		linkForm = "https://www.wopta.it/it/multi-rischio/"
-
+	case "gap":
+		name = "GAP"
```

Comment:
let linkForm with the root of the path -> https://www.wopta.it/it/
in the switch case append the slug
pmi -> multi-rischio/
persona -> infortunio/
life -> vita/
gap -> do not append as there is no page yet